sdk/storage/storage-blob/src/BatchRequest.ts

```diff
@@ -0,0 +1,483 @@
+import {
+  BaseRequestPolicy,
+  deserializationPolicy,
+  generateUuid,
+  HttpHeaders,
+  HttpOperationResponse,
+  RequestPolicy,
+  RequestPolicyFactory,
+  RequestPolicyOptions,
+  WebResource,
+  TokenCredential,
+  isTokenCredential,
+  bearerTokenAuthenticationPolicy,
+  isNode
+} from "@azure/core-http";
+
+import { AnonymousCredential } from "./credentials/AnonymousCredential";
+import { BlobClient, BlobDeleteOptions, BlobSetTierOptions } from "./BlobClient";
+import * as Models from "./generated/src/models";
+import { Mutex } from "./utils/Mutex";
+import { Pipeline } from "./Pipeline";
+import { getURLPath, getURLPathAndQuery, iEqual } from "./utils/utils.common";
+import {
+  HeaderConstants,
+  BATCH_MAX_REQUEST,
+  HTTP_VERSION_1_1,
+  HTTP_LINE_ENDING
+} from "./utils/constants";
+import { SharedKeyCredential } from "./credentials/SharedKeyCredential";
+
+export interface BatchSubRequest {
+  /**
+   * The URL of the resource to request operation.
+   */
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Minor: Same to elsewhere for the newly merged code.

Reply:
Added here.